tests/nested/manual/core20-cloud-init-maas-signed-seed-data/task.yaml

```diff
@@ -58,10 +58,6 @@ prepare: |
   cp "$TESTSLIB"/assertions/developer1.account "$NESTED_FAKESTORE_BLOB_DIR/asserts"
   cp "$TESTSLIB"/assertions/developer1.account-key "$NESTED_FAKESTORE_BLOB_DIR/asserts"
 
-  # modify and repack gadget snap to add a defaults section and use our own
-  # prepare-device hook to use the fakedevicesvc
```

Comment:
I think comment should stay?

Reply:
yes please keep the comment

Reply:
Comment added back, thanks for the review